src/support/steps/lcp-beacon-script.ts

```diff
@@ -29,16 +29,21 @@ const [actual, expected]: [LcpData, LcpData] = [{}, {}];
 Given('I visit the following urls in {string}', async function (this: ICustomWorld, formFactor: string, dataTable) {
     let sql: string,
         result: string,
-        resultFromStdout: Row[];
+        resultFromStdout: Row[],
+        viewPortWidth: number = 1350,
+        viewPortHeight: number = 940;
 
     // Set page to be visited in mobile.
     if ( formFactor === 'mobile' ) {
-        await this.page.setViewportSize({
-            width: 360,
-            height: 640,
-        });
+        viewPortWidth = 412;
+        viewPortHeight = 823;
```

Comment:
Umm the screen size won't be working with our acceptance criterias. 
> We should bail-out when:

> (mobile) We have a screen bigger than RUCSS mobile size (393 x 830) (but still WP Rocket generates the mobile cache for it)
> (desktop) We have a screen smaller than RUCSS desktop size (1920x1080)

It means for desktop we should have minimum `1920x1080` 
And for mobile maximum `393 x 830`

Reply:
I'm editing my last comment due to change of specification yesterday: 
For desktop, we should have minimum 1600x700
and for Mobile, we should have maximum 393x830